### DISCUSSION TITLE
Сделать доступ к поиску нагляднее

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/platform#871] [Идеи]
Сейчас для того, чтобы из статьи попасть в поиск, необходимо нажать на три точки в правом верхнем углу, либо нажать на `/`, что не очень очевидно, и многие пользователи говорят, что им приходится возвращаться на главную, чтобы добраться до поиска. 

В этой дискуссии обсуждаем предложения по упрощению доступа к поиску.

**Comment by monochromer** [comment]
Я бы переименовал дискуссию во что-нибудь типа "Сделать доступ к поиску **_нагляднее_**", иначе можно подумать, что нажатие одной клавиши - это сложно.

**Reply by furtivite** [reply]
Переименовал =)

**Comment by nlopin** [comment]
Идеи
* зафигачить маленькое поле поиска рядом со слешом
* Написать «Поиск» рядом со слешом

<img width="1476" alt="image" src="https://user-images.githubusercontent.com/1469636/154656633-0b2c5540-4dfe-4b4b-b348-11ef8c3b6417.png">

**Reply by igsekor** [reply]
Мне очень нравится, как сейчас сделано на сайте (извините). Не понятно совершенно, но очень круто, когда начинаешь этим пользоваться. В момент принятия дизайна, у нас была дискуссия, которая закончилась тем, что мы собирались посмотреть всё на практике и потом улучшать. Я сам пользоваться стал этим меню совсем недавно, но это для меня оказалось крайне удобным вариантом. Три точки при этом отвечают требованиям Material Design (я понимаю, что это скорее антипаттерн сейчас в интерфейсах). Но кажется, что это действительно не очень хорошее решение, когда им не пользуешься изначально. Может сфокусировать иначе внимание на этой кнопке?

Мы говорили о статистике использования. Решили, что важно понять, насколько это стоит нашего первостепенного внимания. Статистика за две недели показала, что этой кнопкой пользуются 3% пользователей. Наверное, это может говорить о двух противоположных направлениях:

- либо читатели не пользуются, потому что не знают об этом
- либо читателям это вообще не надо

Согласен с Колей, что это могло бы быть поле поиска. Есть и другой вариант с кнопкой бургера (в основном принятый паттерн или антипаттерн) вместо точек, который уже был озвучен, но не был в нашей дискуссии тут.

В качестве дополнительных идей, чтобы озвучить максимальное их количество, можно рассмотреть ещё и такие:

- сделать выпадающее меню по клику на разделы (добавить кнопку «стрелка вниз» справа от раздела)
- сделать три точки после последнего слэша
- заменить кнопку доступа к меню на кнопки поиска и навигации
